frontend/models/product-list/index.ts

```diff
@@ -50,66 +50,81 @@ export async function findProductList(
    const result = await strapi.getProductList({
       filters,
    });
+
    const productList = result.productLists?.data?.[0]?.attributes;
-   if (productList == null) {
+
+   const deviceTitle =
+      productList?.deviceTitle ?? filters.deviceTitle?.eq ?? '';
+   const deviceWiki = deviceTitle ? await fetchDeviceWiki(deviceTitle) : null;
+
+   if (productList == null && deviceWiki == null) {
       return null;
    }
-   const productListImageAttributes = productList.image?.data?.attributes;
 
-   const deviceWiki = productList.deviceTitle
-      ? await fetchDeviceWiki(productList.deviceTitle)
+   const handle = productList?.handle ?? '';
+   const parents =
+      productList?.parent ??
+      (deviceWiki?.ancestors
+         ? convertAncestorsToStrapiFormat(deviceWiki.ancestors)
+         : null);
+   const type = productList?.type ?? deviceWiki?.namespace;
```

Comment:
I think this type wants to be one of the productList type enums. So this should probably fall back to a "device parts" type.

Reply:
we only use type for `getProductListType()` and that has a default in its switch so im just going to ditch the const and have the conditional as the parameter.